src/flightcheck/pipes/Desktop/docker/Dockerfile

```diff
@@ -0,0 +1,23 @@
+# Houston appdata docker file
+# Builds an ubuntu base with desktop-file-utils for validating desktop files
+# Build with `docker build -t flightcheck-desktop-util .`
+#
+# TODO: Move this to a unique repository?
+#
+# Version: 1.0.0
+
+FROM ubuntu:xenial
```

Comment:
I am just using what all the other docker files are using. I can switch them all out for `elementary/docker:loki` instead